build.gradle

```diff
@@ -264,7 +264,6 @@ subprojects {
       java {
         srcDir file('src/referenceTest/java')
       }
-      resources.srcDir file('src/referenceTest/resources')
```

Comment:
I'd keep this - it doesn't matter if the directory doesn't exist and it's a standard pattern to have the `resources` dir work.

Reply:
Agreed. Even though the source set doesn't make use of it, it should probably be there to make the lives of future us easier. Will re-add in my next commit.